pyproject.toml

```diff
@@ -14,6 +14,7 @@ requires-python = ">=3.10"
 dependencies = [
     "langchain-core>=0.3.36,<0.4",
     "mcp>=1.4.1,<1.7",
+    "websockets>=15.0.1",
```

Comment:
let's remove this, we can try importing `websockets` inside the `connect_to_server_via_websocket` and re-raise import error saying that users need to install `mcp[ws] or websockets` to enable this